config/initializers/sentry.rb

```diff
@@ -31,6 +31,8 @@
       end
     when /delayed_job/
       0.001
```

Comment:
```suggestion
      end
```

Reply:
We still need this whilst we are running delayed job and sidekiq in parallel